tests/src/Functional/EtherpadPluginTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\collaborative_editors\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Core\Url;
+
+/**
+ * Check if our defined routes are working correctly or not.
+ *
+ * @group collaborative_editors
+ * @runTestsInSeparateProcesses
+ * @preserveGlobalState disabled
+ */
+class EtherpadPluginTest extends BrowserTestBase {
+
+  /**
+   * An admin user used for this test.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $adminUser;
+
+  /**
+   * The permissions of the admin user.
+   *
+   * @var string[]
+   */
+  protected $adminUserPermissions = [
+    'access administration pages',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'collaborative_editors',
+    'ce_etherpad'
```

Comment:
It shouldn't be required here, if you require ce_etherpad module then these tests should be defined within ce_etherpad module not in the main module.

Reply:
Since I'm going to add more tests of Etherpad for which I'll require "ce_etherpad"
Hence I'm moving it in "ce_etherpad" module